flink-connectors/flink-connector-hbase/src/main/java/org/apache/flink/connector/hbase/source/AbstractTableInputFormat.java

```diff
@@ -99,6 +108,14 @@
 	 */
 	public abstract void configure(Configuration parameters);
 
+	protected void setSerializedConfig(org.apache.hadoop.conf.Configuration hConf) {
```

Comment:
I think we don't need such method, we can directly call `HBaseConfigurationUtil.serializeConfiguration` in the construct?